content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -127,19 +130,26 @@ ls -l
 
 The output shows that `testfile` has group ID 2000, which is the value of `fsGroup`.
 
-```shell
+```none
 -rw-r--r-- 1 1000 2000 6 Jun  6 20:08 testfile
 ```
 
 Run the following command:
 
 ```shell
-$ id
+id
+```
+
+The output is similar to this:
+
+```none
 uid=1000 gid=3000 groups=2000
 ```
-You will see that gid is 3000 which is same as `runAsGroup` field. If the `runAsGroup` was omitted the gid would
-remain as 0(root) and the process will be able to interact with files that are owned by root(0) group and that have
-the required group permissions for root(0) group.
+
+You will see that gid is 3000 which is same as `runAsGroup` field.
```

Comment:
nit: From the output, you can see that `gid` is 3000 which is the same as the `runAsGroup` field?
or
The `gid` is 3000 which is the same [value] as the `runAsGroup` field.

The rest of the paragraph could use a bit more tidying, but can leave it for another PR.